addons/rose/package.json

```diff
@@ -57,18 +57,18 @@
     "@ember/test-helpers": "^2.8.1",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
-    "@storybook/addon-a11y": "^6.3.12",
-    "@storybook/addon-actions": "^6.3.10",
-    "@storybook/addon-controls": "^6.5.12",
-    "@storybook/addon-docs": "~6.3.8",
-    "@storybook/addons": "^6.3.9",
-    "@storybook/api": "^6.3.12",
-    "@storybook/cli": "^6.3.10",
-    "@storybook/components": "^6.3.12",
-    "@storybook/core-events": "^6.3.0",
-    "@storybook/ember": "6.4.0",
+    "@storybook/addon-a11y": "~6.4.0",
+    "@storybook/addon-actions": "~6.4.0",
+    "@storybook/addon-controls": "~6.4.0",
```

Comment:
This dependency is "lowered" to align with all the `@storybook` dependencies being at `~6.4.0`.

Reply:
Yeah I think this is better, I've noticed there's been incompatibilities between different minor versions of storybook addons so we should just make sure they're consistent